src/frontend/js/hooks/useDashboardAddressForm.tsx

```diff
@@ -108,7 +108,7 @@ export const useDashboardAddressForm = (address?: Address) => {
       />
       <CountrySelectField
```

Comment:
I was wondering if we must turn all input of addresses forms to required because atm the `*` is never shown. We only display errors afterwards when trying to submit which can be misleading.